### PR TITLE
added http 0.13.0 support

### PR DESCRIPTION
**Pull request by abdalkhalik**

### CODE REVIEW

**Comment by bhavin-concetto** [comment]
@abdalkhalik can I merge my PR with you as look like you have not migrated to Null Safety?

**Comment by abdalkhalik** [comment]
> merge my PR

Yea, sure

**Comment by abdalkhalik** [comment]
@bhavin-concetto  Yea, sure

File: lib/src/network_util.dart [hunk 1]
```diff
@@ -47,7 +47,7 @@ class NetworkUtil {
 
     String url = uri.toString();
```
Comment (namchuai) [inline lib/src/network_util.dart:48]: I think it's safely to remove this line and the comment below.

**Comment by Dammyololade** [comment]
Already fixed and deployed, thanks.

**Pull request closed without merging**